itest/godot/PropertyTemplate.gd

```diff
@@ -0,0 +1,221 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Couldn't 3/4 of this file be generated in `itest/build.rs`?
(Everything except the code below `# Exporting with custom hints`)

Reply:
probably, i'll look into it.